serverless.yml

```diff
@@ -1,12 +1,14 @@
 service: cloudflare-scanner
 
+frameworkVersion: ^3.7
+
 provider:
   name: aws
   runtime: provided.al2
   stage: prod
-  timeout: 30
+  timeout: 300
   versionFunctions: false
-  memorySize: 512
+  memorySize: 128
```

Comment:
I am guessing we weren't coming anywhere close to 512?

Reply:
Nope. I tried to figure out how much, but our biggest personnel-sync Lambda is configured for 128MB and it's been working for years.